### PR TITLE
Flatzebra: Added devel package for 0.1.6

### PR DESCRIPTION
**Pull request by lich**

### CODE REVIEW

File: dev-games/flatzebra/flatzebra-0.1.6.recipe [hunk 1]
```diff
@@ -42,18 +51,19 @@ BUILD_PREREQUIRES="
...
-	rm $libDir/libflatzebra-0.1.la
-	rm -rf $prefix/include \
-		$prefix/share
+	fixPkgconfig
+
+	#move the flatzebra headers to the place specified in pkgconfig
+	mv $includeDir/flatzebra-0.1/flatzebra $includeDir
+	rm -rf $includeDir/flatzebra-0.1
+
+	rm $-rf $libDir/libflatzebra-0.1.la
```
Comment (diversys) [inline dev-games/flatzebra/flatzebra-0.1.6.recipe]: Looks like a copy paste error

Comment (lich) [inline dev-games/flatzebra/flatzebra-0.1.6.recipe]: What do you mean?

Comment (diversys) [inline dev-games/flatzebra/flatzebra-0.1.6.recipe]: rm $

Comment (lich) [inline dev-games/flatzebra/flatzebra-0.1.6.recipe]: Must have made it during style fixes. Ill upload new version in a moment.

**Pull request merged**